### PR TITLE
Adds anchors to Learn Page

### PR DESCRIPTION
**Pull request by Nick-Tallguy**
add ability to link to all h2 headers in Learn section. #1125 refers

### CODE REVIEW

**Comment by bgirardot** [comment]
@Nick-Tallguy Hi Nick, thank you for the PR. In testing this I do not see any names for the anchors so there is still no way to direct link to a point mid page.

Further testing reveals it was tricky to figure out where best to put the anchors, but I think I have it figured out. 

I know you are busy so I will leave it up to you: I can close this PR and use the file I created testing this pr and just make another PR from my fork, or you can edit your branch as indicated in the graphic below, that will update this PR automatically if you push changes to your branch.

Basically I moved your `<a href=""></a>` to just be a named anchor (`<a...>` tag) between the major `<div>` that hold each section on the learn page.

2018-06-30_00h13_49

Please let me know nick as I am eager to get your changes into the TM, either as your PR or mine, whatever you prefer.

Cheers
Blake

**Comment by Nick-Tallguy** [comment]
Hi Blake,

I don't think I will be able to get to this again for at least 5 days.

 I would just be happy to see this done.

Sorry I didn't get it right first time.

Regards

Nick (OSM=Tallguy)
my phone is responsible for any spelling mistakes!

**Comment by bgirardot** [comment]
@Nick-Tallguy I am going to merge this in and then I'll make a separate pull request to finalize it.
